lib/defaults.js

```diff
@@ -0,0 +1,60 @@
+var defaults = {
+  jquery: {
+    style: 'jquery',
+    noSemver: false,
```

Comment:
I don't think we should add options that are disabled by default, and we don't have a usecase for disabling them.